<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuracion</class>
 <widget class="QDialog" name="Configuracion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuraci칩n</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabConf">
      <attribute name="title">
       <string>Principal</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Cambiar por</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QComboBox" name="cbCentros"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Centro actual</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3">
          <widget class="QLineEdit" name="edtCentro">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Copia de seguridad de los datos</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Usuario</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Contrase침a</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Servidor</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Nombre de la base de datos</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="QLineEdit" name="ledtUserBD"/>
         </item>
         <item row="7" column="1" colspan="2">
          <widget class="QLineEdit" name="ledtPassBD">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="8" column="1" colspan="2">
          <widget class="QLineEdit" name="ledtServerBD"/>
         </item>
         <item row="9" column="1" colspan="2">
          <widget class="QLineEdit" name="ledtNameBD"/>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Datos conexi칩n base de datos</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="btnBackup">
           <property name="text">
            <string>Copia de seguridad</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="ledtBinBackup">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QToolButton" name="btnBinBachup">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabUser">
      <attribute name="title">
       <string>Usuarios</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1" colspan="4">
          <widget class="QTableWidget" name="tableUsuarios">
           <property name="editTriggers">
            <set>QAbstractItemView::DoubleClicked</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectItems</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Usuarios</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QToolButton" name="btnAnyadirUser">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="iconos.qrc">
               <normaloff>:/iconos/iconos/Add.ico</normaloff>:/iconos/iconos/Add.ico</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnEliminarUser">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="iconos.qrc">
               <normaloff>:/iconos/iconos/Delete.ico</normaloff>:/iconos/iconos/Delete.ico</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="btnResPass">
           <property name="text">
            <string>Restablecer contrase침a</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbCentros</tabstop>
  <tabstop>edtCentro</tabstop>
 </tabstops>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections/>
</ui>